test/fw/bin/pbs_snapshot

```diff
@@ -91,12 +91,21 @@ Usage: pbs_snapshot -o <path to output tar file> [OPTION]
 
 if __name__ == '__main__':
 
-    snap_args = {"diag": None, "out_dir": None, "hostname": None,
-                 "log_level": "INFOCLI2", "num_acct_logs": 30,
-                 "num_service_logs": 5, "additional_hosts": None,
-                 "map_file": "obfuscate.map", "anonymize": False}
+    # Arguments to PBSSnapUtils
+    out_dir = None
+    diag_path = None
+    server_host = None
+    log_level = "INFOCLI2"
+    acct_logs = 30  # Capture 30 days of accounting logs by default
+    srvc_logs = 5   # Capture 5 days of service logs by default
+    additional_hosts = None
+    map_file = "obfuscate.map"
```

Comment:
Where is obfuscate.map put by default?  Is it in the snap dir?  I suggest putting it somewhere else.  It's too easy for someone to tar up the snap dir and not notice the map is in there.  That defeats the purpose of anonymizing the data.